buddyboss/buddyboss-platform
# Fixed While uploading Profile photo or cover photo, it should update instantly on page

### All Submissions:


Fixes #1655 

### How to test the changes in this Pull Request:

1. Go to Profile page and update profile image
2. It should update profile image everywhere seen on page
3. Go to Group page and update cover photo
4. It should update cover photo  everywhere seen on page

### Other information:
